packages/main/src/components/FilterBar/FilterBar.test.tsx

```diff
@@ -500,7 +500,7 @@ describe('FilterBar', () => {
     const dialogGoButton = screen.getAllByText('Go')[1];
     fireEvent.click(dialogGoButton);
     expect(onGo).toHaveBeenCalledTimes(2);
-    expect(onFiltersDialogSave).toHaveBeenCalledTimes(1);
+    expect(onFiltersDialogSave).toHaveBeenCalledTimes(2);
```

Comment:
why did those counters increase?

Reply:
Before, onGo just closed the dialog and din't update the values inside the FilterBar. Now, when the "Go" button is clicked, it's also saving the values and therefore is also calling the corresponding callback.